kie-wb-tests/kie-wb-tests-gui/src/test/java/org/kie/wb/selenium/ui/ProjectAuthoringIntegrationTest.java

```diff
@@ -33,19 +35,24 @@
 
     private static final Logger LOG = LoggerFactory.getLogger( ProjectAuthoringIntegrationTest.class );
 
+    private HomePerspective home;
+
+    @Before
+    public void setup() {
```

Comment:
Does it make sense to import mortgage project before every test in this class? The common mistake people do when writing selenium tests is writing them as if they were unit tests. From QE's experience every 10 seconds added to the test execution time makes the tests less valuable and people more likely to skip wasting time on their maintenance ;-)